scripts/proto-doc-gen.sh

```
@@ -0,0 +1,79 @@
+if [ "$ec" != '0' ]; then
+  printf 'Base directory . = %s\n' "`pwd`" >&2
+  exit "$ec"
+fi
+
+# Generate the docs/proto-docs.md file.
+protoc \
+  -I "proto" \
+  -I "third_party/proto" \
+  --doc_out=./docs \
+  --doc_opt=./docs/protodoc-markdown.tmpl,proto-docs.md \
+  $( find "proto" -name '*.proto' )
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)